e2e/test/scenarios/joins/reproductions/22859-multi-nested-joins-wrong-aliasing.cy.spec.js

```diff
@@ -31,45 +31,53 @@ const questionDetails = {
 
 describe("issue 22859 - multiple levels of nesting", () => {
   beforeEach(() => {
-    cy.intercept("POST", "/api/card").as("saveQuestion");
-
     restore();
     cy.signInAsAdmin();
 
     cy.createQuestion(questionDetails, { wrapId: true, idAlias: "q1Id" });
 
     // Join Orders table with the previously saved question and save it again
-    openOrdersTable({ mode: "notebook" });
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.findByText("Join data").click();
-
-    popover().within(() => {
-      cy.findByText("Sample Database").click();
-      cy.findByText("Raw Data").click();
-      cy.findByText("Saved Questions").click();
-      cy.findByText(questionDetails.name).click();
+    cy.get("@q1Id").then(id => {
+      const nestedQuestionDetails = {
+        name: "22859-Q2",
+        query: {
+          "source-table": ORDERS_ID,
+          joins: [
+            {
+              fields: "all",
+              alias: `Question ${id}`,
+              condition: [
+                "=",
+                ["field", ORDERS.PRODUCT_ID, { "base-type": "type/Integer" }],
+                [
+                  "field",
+                  REVIEWS.PRODUCT_ID,
+                  {
+                    "base-type": "type/Integer",
+                    "join-alias": `Question ${id}`,
+                  },
+                ],
+              ],
+              "source-table": `card__${id}`,
+            },
+          ],
+          limit: 5,
```

Comment:
5 or 10k rows don't make this issue more or less likely to happen.
So, why not stick with 5 and ease the load on the runner?